airflow-core/src/airflow/ui/src/i18n/locales/de/admin.json

```
@@ -1,6 +1,161 @@
+  },
+  "formActions": {
+    "reset": "Zurücksetzen",
+    "save": "Speichern"
+  },
+  "plugins": {
+    "columns": {
+      "source": "Quelle"
+    },
+    "importError_one": "Fehler beim Laden eines Plug-in",
+    "importError_other": "Fehler beim Laden von Plug-ins",
+    "searchPlaceholder": "Nach Datei suchen"
+  },
+  "pools": {
+    "add": "Pool hinzufügen",
+    "deferredSlotsIncluded": "Delegierte Tasks einschließen",
+    "delete": {
+      "title": "Pool löschen",
+      "warning": "Dies wird alle Metadaten zu diesem Pool entfernen. Damit kann es eine Auswirkung auf Tasks geben, die diesen Pool nutzen."
+    },
+    "edit": "Pool bearbeiten",
+    "form": {
+      "checkbox": "Delegierte Tasks sollen in die Berechnung der Pool-Belegung mit einbezogen werden",
+      "description": "Beschreibung",
+      "includeDeferred": "Delegierte Tasks",
+      "nameMaxLength": "Der Name kann maximal 256 Zeichen lang sein",
+      "nameRequired": "Ein Name ist erforderlich",
+      "slots": "Plätze im Pool"
+    },
+    "noPoolsFound": "Keine Pools gefunden",
+    "pool_one": "Pool",
+    "pool_other": "Pools",
+    "searchPlaceholder": "Pools suchen",
+    "sort": {
+      "asc": "Name (A-Z)",
+      "desc": "Name (Z-A)",
+      "placeholder": "Sortierung"
+    }
+  },
+  "providers": {
+    "columns": {
+      "packageName": "Paketname",
+      "version": "Version"
+    }
+  },
+  "variables": {
+    "add": "Variable hinzufügen",
+    "columns": {
+      "isEncrypted": "Wert ist verschlüsselt"
+    },
+    "delete": {
+      "deleteVariable_one": "1 Variable löschen",
+      "deleteVariable_other": "{{count}} Variablen löschen",
+      "firstConfirmMessage_one": "Die folgende Variable wird gelöscht:",
+      "firstConfirmMessage_other": "Die folgenden Variablen werden gelöscht:",
+      "title": "Variable löschen",
+      "tooltip": "Ausgewählte Variablen löschen"
+    },
+    "edit": "Variable bearbeiten",
+    "export": "Exportieren",
+    "exportTooltip": "Ausgewählte Variablen exportieren",
+    "form": {
+      "invalidJson": "JSON ist ungültig",
+      "keyMaxLength": "Der Name kann maximal 250 Zeichen lang sein",
+      "keyRequired": "Name ist erforderlich",
+      "valueRequired": "Wert ist erforderlich"
+    },
+    "import": {
+      "button": "Importieren",
+      "conflictResolution": "Konfliktlösung für Import auswählen",
+      "errorParsingJsonFile": "JSON-Datei kann nicht gelesen werden: Die Datei muss Schlüssel-/Wertpaare enthalten (z.B., {\"name\": \"wert\", ...}).",
+      "options": {
+        "fail": {
+          "description": "Import abbrechen, wenn eine Variable bereits besteht.",
+          "title": "Abbrechen"
+        },
+        "overwrite": {
+          "description": "Variable überschreiben, die schon vor dem Import besteht.",
+          "title": "Überschreiben"
+        },
+        "skip": {
+          "description": "Bestehende Variablen werden übersprungen (und nicht verändert).",
```

```suggestion
          "description": "Variable überspringen, wenn eine Variable mit dem gleichen Namen bereits existiert (der Wert der bestehenden Variable wird nicht verändert).",
```

Mostly a style/clarification suggestion to make it more explicit. I think 99% of users understand these kind of import options already, but for beginners I think it makes sense to spell it out a little more :)

---

Yeah, I think I was a bit tooo brief. The longer description is better!